draft-bishop-httpbis-http2-additional-certs.md

```diff
@@ -822,12 +831,14 @@ registered by this document.
 | CERTIFICATE_REVOKED     | 0xERROR-TBD3 | {{errors}}              |
 | CERTIFICATE_EXPIRED     | 0xERROR-TBD4 | {{errors}}              |
 | CERTIFICATE_GENERAL     | 0xERROR-TBD5 | {{errors}}              |
+| NEED_DIFFERENT_CERT     | 0xERROR-TBD6 | {{errors}}              |
```

Comment:
Do we need an error code that indicates the peer to provide a new certificate on a new connection?

My understanding is that the draft expects an endpoint to remember all the certificate that the peer has sent in the past. I wonder how the number of the certificate can be capped, and/or, how an endpoint should notify the peer that it has reached the limit.